rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md

```diff
@@ -1,21 +1,19 @@
 ---
 type: rule
-archivedreason: 
 title: Do you use the Code Health Extensions in Visual Studio?
-guid: b0048e8d-b6bb-4af3-9397-a9af6fac1a88
 uri: do-you-use-the-code-health-extensions-in-visual-studio
-created: 2017-03-09T22:11:25.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Danijel Malik
-  url: https://ssw.com.au/people/danijel-malik
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
```

Comment:
@bradystroud do these spaces break the frontmatter

Reply:
@bradystroud, should I return these back to how they were before?

Reply:
No - they are fine